hack/tests/ci-prepare.sh

```diff
@@ -83,7 +83,9 @@ make secrets
 . ./secrets/secret
 export AZURE_CLIENT_ID="$AZURE_CI_CLIENT_ID"
 export AZURE_CLIENT_SECRET="$AZURE_CI_CLIENT_SECRET"
+export AZURE_AAD_CLIENT_ID="$AZURE_AAD_CI_CLIENT_ID"
+export AZURE_AAD_CLIENT_SECRET="$AZURE_AAD_CI_CLIENT_SECRET"
```

Comment:
This will break existing tests. Need an `lgtm` and `hold` and will sync the merge